### ISSUE TITLE
Accessibility fixes: Manage IDs

### ISSUE THREAD

**Issue by mariagould** [CDLUC3/ezid#545]
The following accessibility issues were identified on the Manage IDs page:

- [x] Remove the “Read more/Read less" option to expand and collapse search tips. Show all tips by default.  

<img width="902" alt="Screenshot 2024-01-09 at 08 34 31" src="https://github.com/CDLUC3/ezid/assets/45110557/8b69fc62-29e6-4e80-9fe3-b1efaf751258">

- [x] The identifiers table has redundant links. All metadata fields are links, but they all go to the same identifiers page. Only the content in the "Identifier" column should have a link.

<img width="1028" alt="Screenshot 2024-01-09 at 08 33 35" src="https://github.com/CDLUC3/ezid/assets/45110557/c7714f79-1bc7-4059-ae28-416f93d707ad">

**Comment by adambuttrick** [comment]
@sfisher Reviewed these on dev, but neither fix appeared to be present.

**Comment by sfisher** [comment]
I'll must not have merged everything.  I'll look into it. :-)

**Comment by adambuttrick** [comment]
@sfisher I cleared everything in the browser and this now displays fine! Apologies for the cached confusion.

**Comment by adambuttrick** [comment]
Included in https://github.com/CDLUC3/ezid/releases/tag/v3.3.2.
